src/client/sandbox/electron.js

```diff
@@ -30,6 +53,8 @@ export default class ElectronSandbox extends SandboxBase {
             vm.runInContext      = ElectronSandbox._createFnWrapper(vm, nativeMethods.runInContext);
             vm.runInNewContext   = ElectronSandbox._createFnWrapper(vm, nativeMethods.runInNewContext);
             vm.runInThisContext  = ElectronSandbox._createFnWrapper(vm, nativeMethods.runInThisContext);
+
+            ElectronSandbox._fixModuleName(window);
```

Comment:
I think `_fixModuleName` is bad name for method. We should use `_overrideElectronPaths` or sth like that. @LavrovArtem @churkin What do You think?